src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carCostDay = 40;
+  const rentDays = days;
+
+  if (rentDays > 2) {
+    if (rentDays > 6) {
```

Comment:
You don't need to nest this if statement inside the other one, it's redundant, move it up the code